<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Catalog Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>67</y>
     <width>701</width>
     <height>418</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTreeView" name="catalogView" >
      <property name="contextMenuPolicy" >
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionOpenInEditor" />
    <addaction name="actionFindInFiles" />
    <addaction name="action_Replace_in_files" />
    <addaction name="separator" />
    <addaction name="actionReload" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menu_Tools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSpellcheck" />
    <addaction name="actionStatistics" />
    <addaction name="separator" />
    <addaction name="actionConfigure" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Edit" />
   <addaction name="menu_Tools" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>485</y>
     <width>701</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>25</y>
     <width>701</width>
     <height>42</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Main toolbar</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenInEditor" />
   <addaction name="actionStatistics" />
   <addaction name="actionSpellcheck" />
   <addaction name="actionConfigure" />
   <addaction name="actionQuit" />
  </widget>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/application-exit.png</normaloff>:/icons/icons/application-exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip" >
    <string>Exit application</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionFindInFiles" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/file-find.png</normaloff>:/icons/icons/file-find.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find in files...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Replace_in_files" >
   <property name="text" >
    <string>&amp;Replace in files...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionReload" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/view-refresh.png</normaloff>:/icons/icons/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>R&amp;eload</string>
   </property>
   <property name="toolTip" >
    <string>Reload catalog tree</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About...</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About &amp;Qt...</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionSpellcheck" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/tools-check-spelling.png</normaloff>:/icons/icons/tools-check-spelling.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Spellcheck...</string>
   </property>
   <property name="toolTip" >
    <string>Check spelling</string>
   </property>
   <property name="statusTip" >
    <string>Check spelling of the catalog</string>
   </property>
  </action>
  <action name="actionStatistics" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/document-properties.png</normaloff>:/icons/icons/document-properties.png</iconset>
   </property>
   <property name="text" >
    <string>S&amp;tatistics...</string>
   </property>
  </action>
  <action name="actionOpenInEditor" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/document-open.png</normaloff>:/icons/icons/document-open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open in Editor</string>
   </property>
   <property name="iconText" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="icon" >
    <iconset resource="cm4.qrc" >
     <normaloff>:/icons/icons/configure.png</normaloff>:/icons/icons/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configure...</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="cm4.qrc" />
 </resources>
 <connections/>
</ui>
